<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndTimeline</class>
 <widget class="QMainWindow" name="WndTimeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Katapulto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/imgs/catapult</normaloff>:/imgs/catapult</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <widget class="QPushButton" name="btnTimeline">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#050; 
background-color:#fed; 
border:1px solid; 
border-color: #696 #363 #363 #696;</string>
        </property>
        <property name="text">
         <string>&amp;Friends</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMentions">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#050; 
background-color:#fed; 
border:1px solid; 
border-color: #696 #363 #363 #696;</string>
        </property>
        <property name="text">
         <string>&amp;Mentions</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+M</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDirects">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#050; 
background-color:#fed; 
border:1px solid; 
border-color: #696 #363 #363 #696;</string>
        </property>
        <property name="text">
         <string>&amp;Directs</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+D</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="fraUpdate">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblUpdateTitle">
           <property name="text">
            <string>Digite o novo post</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCount">
           <property name="text">
            <string>0</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TextUpdate" name="txtUpdate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrTimeline">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrTimelineContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>284</width>
         <height>259</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionConfigurar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menu_Arquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfigurar">
   <property name="text">
    <string>&amp;Configurar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sai&amp;r</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>&amp;Update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextUpdate</class>
   <extends>QPlainTextEdit</extends>
   <header>TextUpdate.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
